src/enzo/Make.mach.niagara

```diff
@@ -0,0 +1,127 @@
+#=======================================================================
+#
+# FILE:        Make.mach.niagara
+#
+# DESCRIPTION: Makefile settings for Niagara cluster

```

Comment:
Add where this is located. Scinet? I would actually rename this file to include the location, like `Make.mach.scinet-niagara`